docs/auto-instrumentation-install.md

```diff
@@ -371,22 +371,18 @@ Below is a breakdown of the main and related components involved in auto-instrum
+_Note: Native OpenTelemetry instrumentation libraries are owned and maintained by the OpenTelemetry Community. Splunk provides best effort support for issues related to these libraries._
+
+| Instrumentation Language | Distribution   | Feature Gate Name                                     | Feature Gate Default Value | Status         | Splunk Content Compatibility | Code Repository                                                     | Source URL                                                  | Image Repository                                                               |
+|--------------------------|----------------|-------------------------------------------------------|----------------------------|----------------|------------------------------|---------------------------------------------------------------------|-------------------------------------------------------------|--------------------------------------------------------------------------------|
+| Java                     | Splunk         | `operator.autoinstrumentation.java`                   | Enabled                    | Stable         | Fully Compatible             | [Link](https://github.com/signalfx/splunk-otel-java)                | https://github.com/signalfx/splunk-otel-java                | ghcr.io/signalfx/splunk-otel-java/splunk-otel-java                             |
+| NodeJS                   | Splunk         | `operator.autoinstrumentation.nodejs`                 | Enabled                    | Stable         | Fully Compatible             | [Link](https://github.com/signalfx/splunk-otel-js)                  | https://github.com/signalfx/splunk-otel-js                  | ghcr.io/signalfx/splunk-otel-java/splunk-otel-js                               |
+| DotNet                   | Splunk         | `operator.autoinstrumentation.dotnet`                 | Enabled                    | Stable         | Fully Compatible             | [Link](https://github.com/signalfx/splunk-otel-dotnet)              | https://github.com/signalfx/splunk-otel-dotnet              | ghcr.io/signalfx/splunk-otel-java/splunk-otel-dotnet                           |
+| Python                   | Splunk         | `operator.autoinstrumentation.python`                 | Enabled                    | Experimental   | Partially Compatible         | [Link](https://github.com/open-telemetry/opentelemetry-python)      | https://github.com/open-telemetry/opentelemetry-python      | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python       |
+| ApacheHttpD              | OpenTelemetry  | `operator.autoinstrumentation.apache-httpd`           | Enabled                    | Experimental   | Partially Compatible         | [Link](https://github.com/open-telemetry/opentelemetry-cpp-contrib) | https://github.com/open-telemetry/opentelemetry-cpp-contrib | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd |
```

Comment:
Based on our current stance, I suggest disabling python and apache httpd by default. We will fast follow with our distribution of Python next quarter and can enable auto instrumentation for it thereafter.

Reply:
Done